techniques/general/G144.html

```diff
@@ -51,69 +49,68 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
-<!-- 日本語訳されるまでの一時的な注記 -->
-<div class="untranslated-note"><p>このWCAG 2.2 テクニック集の日本語訳は作業中となっています。WCAG 2.1 達成方法集の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Techniques/">WCAG 2.1 達成方法集</a> ]</p></div>
-<h1><span>Technique G144:</span>Ensuring that the Web Page contains another CAPTCHA serving the same purpose using a different modality</h1>
+
+<h1><span>テクニック G144:</span>異なるモダリティを用いて同じ目的を果たす、もう一つの CAPTCHA がウェブページに含まれていることを確認する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
   <p>
-    This technique relates to
+    このテクニックは
     
-<a href="https://w3c.github.io/wcag/understanding/non-text-content">1.1.1: Non-text Content</a>
-(<strong>Sufficient</strong>, together with <a href="../general/G143">G143: Providing a text alternative that describes the purpose of the CAPTCHA</a>).</p>
+<a href="https://waic.jp/translations/WCAG22/Understanding/non-text-content">1.1.1: 非テキストコンテンツ</a>
+(<a href="../general/G143">G143: Providing a text alternative that describes the purpose of the CAPTCHA</a>と一緒に<strong>十分なテクニック</strong>) に関連する。</p>
```

Comment:
英単語と日本語の間、スペースを入れました。
```suggestion
(<a href="../general/G143">G143: Providing a text alternative that describes the purpose of the CAPTCHA</a> と一緒に<strong>十分なテクニック</strong>) に関連する。</p>
```